deploy/scripts/src/common/utils/aws/types.d.ts

```diff
@@ -0,0 +1,13 @@
+// AWS CLI Credential Output Format https://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html#examples
+export interface AssumeRoleOutput {
```

Comment:
Rather than defining the type I'd recommend importing the AWS Client STS library `@aws-sdk/client-sts` and use `AssumeRoleCommandOutput`. Also you don't need to have quotes `'...'` for definitions.

Reply:
Thanks, I've removed the single quotes in the interface definition.

The k6 scripts are not strictly TS/JS where you can important any library. Instead we can only import `k6` libraries and small files written in TS/JS. So having a small type definition file for this interface is the simplest solution here.